etc/env/base/eth_sender.toml

```diff
@@ -66,3 +66,4 @@ pricing_formula_parameter_b=1.0005
 internal_l1_pricing_multiplier=0.8
 # Node polling period in seconds.
 poll_period=5
+l1_gas_per_pubdata_byte=17
```

Comment:
this is coming back?

Reply:
Nope! Probably came back in some PR erroneously